examples/standalone/custom_server/CMakeLists.txt

```diff
@@ -2,10 +2,10 @@ cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
 
```

Comment:
This is not part of this PR. Do you mind to remove it ?

Reply:
I noticed it while working on this PR. It could go on its own PR, but I thought it was small enough to put it here, this way we save some CI cycles